doc/source/releases/17.01_announce.rst

```diff
@@ -3,8 +3,50 @@
 January 2017 Galaxy Release (v 17.01)
 ===========================================================
 
+.. include:: _header.rst
 
-Schedule
+Highlights
 ===========================================================
- * Planned Freeze Date: 2017-01-02
- * Planned Release Date: 2017-01-23
+
+**Conda auto initialization is enabled by default**
+  After updating to 17.01 and upon starting Galaxy the Conda package manager will be installed automatically
+  within the Galaxy's workspace. This will enable your Galaxy to install tool dependencies from Conda channels
+  such as `BioConda <https://bioconda.github.io/>`__. Such channels are already being used for new and updated tools by IUC
+  and we suggest adopting this tool dependency resolution method to all tool developers instead of Tool Shed package
+  recipes - which are now considered deprecated.
+  Please see the details at the admin `documentation <https://docs.galaxyproject.org/en/master/admin/conda_faq.html>`__
+
+**New interface for user preferences**
+  User preferences menu has been reworked for clarity and consistency.
+  Thanks to `@anuprulez <https://github.com/anuprulez>`__ and `@bgruening <https://github.com/bgruening>`__.
+  Implemented in `Pull Request 3118`_.
+
+**Support for compressed FASTQ formats**
+  You can now upload compressed (with gzip or bzip2) FASTQ files without them being unpacked automatically (as it was until now).
+  For this to work you have to select the proper type when uploading (e.g. instead of ``fastqsanger`` you select ``fastqsanger.gz`` for compressed files).
+  Existing tools will work as before and future tools will be able to consume archives and save space in your quota.
+  Thanks to `@mvdbeek <https://github.com/mvdbeek>`__.
```

Comment:
`@pvanheus @abretaud @ashok` also contributed commits in that PR,
and I think we should also mention https://github.com/galaxyproject/galaxy/pull/3514 and https://github.com/galaxyproject/galaxy/pull/3510